telemetry/definitions/commonDefinitions.json

```diff
@@ -250,6 +256,41 @@
             "description": "Called when starting a local API Gateway server simulator with SAM. Only called when starting it for long running testing, not for single invokes",
             "metadata": [{ "type": "result" }]
         },
+        {
+            "name": "apprunner_copyServiceUrl",
+            "description": "Copy the service URL"
+        },
+        {
+            "name": "apprunner_createService",
+            "description": "Create an AppRunner service",
+            "metadata": [
+                { "type": "result" },
+                { "type": "appRunnerServiceSource" }
+            ]
+        },
+        {
+            "name": "apprunner_pauseService",
+            "description": "Pause a running AppRunner service",
+            "metadata": [{ "type": "result" }]
+        },
+        {
+            "name": "apprunner_resumeService",
+            "description": "Resume a paused AppRunner service",
+            "metadata": [{ "type": "result" }]
+        },
+        {
+            "name": "apprunner_startDeployment",
+            "description": "Start a new deployment for an AppRunner service",
+            "metadata": [{ "type": "result" }]
+        },
+        {
+            "name": "apprunner_viewApplicationLogs",
+            "description": "View the AppRunner service logs"
```

Comment:
```suggestion
            "description": "View the AppRunner application logs"
```

Reply:
The top level construct is called `service`

Reply:
I know I thought App Runner had logs for the running application plus logs for the service resource.

Reply:
well this is confusing https://docs.aws.amazon.com/apprunner/latest/dg/monitor-cwl.html

service = system logs
application = logs for a service

So changed to match the service naming scheme and updated the descriptions